### PR TITLE
FOLIO-3293: Adding rancher deployment instructions to readme

### PR DESCRIPTION
**Pull request by steveellis**
Recently had to deploy this onto rancher and wanted to document all the steps that were necessary.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,6 +33,24 @@ API provides the following URLs for working with remote storage configurations:
 3. User with name `{{username}}`, password `{{password}}`, remote-storage.all permissions should be created on FOLIO. After that apikey can
    be generated for making calls to Edge CaiaSoft API.
 
+#### Rancher and kubernetes deployment
+1. Check that mod-remote-storage has been installed and has been registered to okapi.
+2. Create a new user named `stagingDirector` in FOLIO. You may also use `diku_admin` for testing and avoid this step.
+3. Create a secret in the rancher cluster. Make the key of this secret `ephemeral.properties` and the value similar to `secureStore.type=Ephemeral tenants=diku diku=diku_admin,admin,stagingDirector`.
```
Comment (julianladisch) [inline README.md:39]: Does this use spaces or line breaks as separators?

Comment (steveellis) [reply README.md:31]: The list above my changes had three spaces inserted in the middle of lines probably to try to make the list be indented. I removed those to make the file more consistent.

**Review by julianladisch — APPROVED** [review]
Looks good, thanks!

File: README.md [hunk 2]
```diff
@@ -20,17 +20,28 @@ API provides the following URLs for working with remote storage configurations:
...
+2. For each tenant using CaiaSoft the corresponding user should be added to the AWS parameter store with key in the following format `{{username}}_{{tenant}}_{{username}}` (where salt and username are the same - `{{username}}`) with value of corresponding `{{password}}` (as Secured String). This user should work as ordinary edge institutional user with the only one difference - his username and salt name are same. By default the value of `{{username}}` is `caiaSoftClient`. It could be changed through `caia_soft_client` parameter of starting module.
+3. User with name `{{username}}`, password `{{password}}`, remote-storage.all permissions should be created on FOLIO. After that apikey can be generated for making calls to Edge CaiaSoft API.
+
+#### Rancher and kubernetes deployment
+1. Check that mod-remote-storage has been installed and has been registered to okapi.
+2. Create a new user named `stagingDirector` in FOLIO. You may also use `diku_admin` for testing and avoid this step.
```
Comment (alekGbuz) [inline README.md:30]: Please  change username from "stagingDirector" to "caiaSoftClient".

File: README.md [hunk 3]
```diff
@@ -20,17 +20,28 @@ API provides the following URLs for working with remote storage configurations:
...
+3. Create a secret in the rancher cluster. Make the key of this secret `ephemeral.properties` and the value similar to `secureStore.type=Ephemeral tenants=diku diku=diku_admin,admin,stagingDirector`.
+4. Add this secret as a volume mount to the workload for the edge module container. Set the mount point of this volume to `\etc\edge`.
+5. Set the `JAVA_OPTIONS` environment variable for the workload to something similar to `-Dsecure_store_props=/etc/edge/ephemeral.properties -Dokapi_url=http://okapi:9130 -Dlog_level=DEBUG -Dstaging_director_client=diku_admin`.
```
Comment (alekGbuz) [inline README.md:33]: Please change -Dstaging_director_client=diku_admin to -Dcaia_soft_client=diku_admin

**Review by alekGbuz — APPROVED** [review]

**Pull request merged**